src/jit/hwintrinsicxarch.cpp

```diff
@@ -460,25 +460,21 @@ GenTree* Compiler::impHWIntrinsic(NamedIntrinsic        intrinsic,
 
     if ((flags & (HW_Flag_OneTypeGeneric | HW_Flag_TwoTypeGeneric)) != 0)
     {
-        assert(baseType != TYP_UNKNOWN);
         // When the type argument is not a numeric type (and we are not being forced to expand), we need to
```

Comment:
Is this comment still correct? I would guess not with the removal of the below assert

Reply:
Thanks, should remove.